stubs/geopandas/geopandas/array.pyi

```diff
@@ -21,11 +21,14 @@ _Array2D: TypeAlias = np.ndarray[tuple[int, int], np.dtype[_ScalarType]]
 _ArrayOrGeom: TypeAlias = GeometryArray | ArrayLike | Geometry
 
 TransformerFromCRS = Transformer.from_crs
+POLYGON_GEOM_TYPES: Final[set[str]]
+LINE_GEOM_TYPES: Final[set[str]]
+POINT_GEOM_TYPES: Final[set[str]]
 
 class GeometryDtype(ExtensionDtype):
     type: ClassVar[type[BaseGeometry]]
     name: ClassVar[str]
-    na_value: float
+    na_value: None
```

Comment:
Possible values ​​of `on_invalid` parameter for `from_wkb` and `from_wkt` functions [have been updated](https://github.com/geopandas/geopandas/pull/3547). a new input values: `Literal["raise", "warn", "ignore", "fix"]`